pom.xml

```diff
@@ -108,6 +108,11 @@
             <artifactId>maven-model</artifactId>
             <version>3.3.9</version>
         </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
```

Comment:
I don't know how common this module is, but there can be issues with other plugins or core providing the same dependency and that could potentially lead to trouble.
If core or any dependant plugin is providing the same you need to shade it or something. For the maven-model dependency my investigation showed that we shouldn't have any dupes, so that was fine. But I don't know about yaml.